pom.xml

```diff
@@ -124,7 +124,7 @@
     <test.output.redirect>true</test.output.redirect>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>1.6</java.version>
-    <hadoop.version>1.0.4</hadoop.version>
+    <hadoop.version>2.6.0</hadoop.version>
```

Comment:
While I personally would love Tachyon to target Hadoop 2.x by default I expect that you will get some pushback on this particular change